source/client/index.js

```diff
@@ -1,7 +1,12 @@
 const React = require('react')
-const withScroll = require('scroll-behavior').default
+const useScroll = require('react-router-scroll/lib/useScroll')
```

Comment:
I'm not the biggest fan of reaching into `lib` dirs... there's no guarantee that the directory structure should be kept.